herd/AArch64Sem.ml

```diff
@@ -1399,6 +1404,14 @@ module Make
                  mop ac ma >>= M.ignore >>= B.next1T
         )
 
+      let lift_kvm dir updatedb mop ma an ii mphy =
```

Comment:
It appears as though `lift_kvm` raises a synchronous fault by default, while `do_lift_kvm` permits passing a fault as a parameter. Seems neat.